samples/subsys/zbus/work_queue/src/main.c

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "messages.h"
+
+#include <stdint.h>
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/sys/util_macro.h>
+#include <zephyr/zbus/zbus.h>
+LOG_MODULE_DECLARE(zbus, CONFIG_ZBUS_LOG_LEVEL);
+
+ZBUS_CHAN_DEFINE(version_chan,	     /* Name */
+		 true,		     /* Read only */
+		 struct version_msg, /* Message type */
+
+		 NULL,		       /* Validator */
+		 ZBUS_OBSERVERS_EMPTY, /* observers */
+		 ZBUS_MSG_INIT(.major = 0, .minor = 1,
+			       .build = 1023) /* Initial value major 0, minor 1, build 1023 */
+);
+
+ZBUS_CHAN_DEFINE(sensor_data_chan,  /* Name */
+		 false,		    /* Read only */
+		 struct sensor_msg, /* Message type */
+
+		 NULL, /* Validator */
+		 ZBUS_OBSERVERS(fast_handler1_lis, fast_handler2_lis, fast_handler3_lis,
+				delay_handler1_lis, delay_handler2_lis, delay_handler3_lis,
+				thread_handler1_sub, thread_handler2_sub,
+				thread_handler3_sub), /* observers */
+		 ZBUS_MSG_INIT(0)		      /* Initial value {0} */
+);
+
+void fh1_cb(struct zbus_channel *chan)
```

Comment:
These should be all static (listener handles, thread handlers etc). Same as in other samples.